apps/server/src/shared/domain/entity/news.entity.ts

```diff
@@ -42,6 +42,10 @@ export abstract class News extends BaseEntityWithTimestamps {
 	@Index()
 	displayAt: Date;
 
+	@Property({ nullable: true })
```

Comment:
We can't use updateAt field because vaule for it is set when we create the news. I will remove this index.

Reply:
Yes when create an update but this is what you want or? You also want to display the information if it is new created.

Reply:
Sorry didn't understood I think. We just want to display update time but this can't be done properly if we read it from updateAt because when we create the fresh news entry it is not null but we have the same value as createdAt field so we need to add additional field on the entity that on entry is undefined/null

Reply:
Created time was already covered before we are just adding update information

Reply:
When i create a new news (without update) and a other open it 20 minutes later, then i expect that it display last update 20 minutes before. This result into avoid the additional update key. A creating is also a change of the values it self. Let say first change. This is the reasons why updateAt directly get a date.

Reply:
@CeEv but I think it's business incorrect to always show the `updatedAt` timestamp as this would result in a sense that all the news has already been updated (every news from now on would display something like "updated 32 seconds ago | created 32 seconds ago" which is simply not true - there were no update.

But okay as a compromise that would still allow us to make it valid from the business logic point of view is that, as you proposed in the last part of your comment, we implement the following algorithm:
- if updatedAt == createdAt - display only the creation date (so we would only see "created 32 seconds ago")
- else - display both dates (so we would only see it when the dates differ, e.g. "updated 15 seconds ago | created 32 seconds ago"

This way the message to the users will be correct from the business logic point of view and we'll be able to get rid of an additional timestamp that would be used only for this purpose.

BTW I'm not sure if it will still be fully valid - I mean as `createdAt` is more like a technical field which might not always reflect business action (maybe there are some cases when the database entity would be updated without the real news message update performed by any user?) so I think we'll have to rely on your knowledge @CeEv and if any error will be spotted (the message will show any business incorrect information as a side effect of any other news modification operation) we'll then discuss it again and maybe introduce a separate field just for that purpose. But until then let's just use the `updatedAt` field @sszafGCA .

Reply:
@CeEv chnaged